shell/platform/darwin/ios/framework/Source/FlutterEngine.mm

```diff
@@ -557,20 +557,6 @@ + (NSString*)generateThreadLabel:(NSString*)labelPrefix {
           threadHostType};
 }
 
-static void SetEntryPoint(flutter::Settings* settings, NSString* entrypoint, NSString* libraryURI) {
```

Comment:
I removed this new indirection since we're no longer filling in the Settings in the Spawn to partially consume it anymore as explained above.